README.md

```diff
@@ -48,12 +48,16 @@ You must set at least one of `user_id_key` and `device_id_key`. They will be use
 #### time_key
 If set, `time_key` will be used to pull out a timestamp field to set as `time` in the Amplitude API request. This can be an array, the first matching key will be used.
 
+####  revenue_properties
+
+`revenue_properties` are a fixed set of properties (e.g., `price`, `quantity`, `revenue_type`) that Amplitude looks for in order to determine which events are revenue events (see the Amplitude HTTP API documentation for more information).
```

Comment:
Should probably also note that these are passed as top-level properties in the API call.